### ISSUE TITLE
Window size is wrong in new tabs

### ISSUE THREAD

**Issue by LogicalJoe** [debrouxl/tilp_and_gfm#34]
When it opens a new tab (not window), the window sizing is wrong as it maintains the original size.  It is not corrected until the window is resized, or is pulled out into a new window.
![Screen Shot 2020-03-05 at 15 33 46](https://user-images.githubusercontent.com/47825483/76023271-d6630200-5ef6-11ea-902a-02cf82d02289.png)
Screenshot should say enough.
This also applies to the "TiLP2" tab as the window is resized but it doesn't acknowledge it.
The setting that affects this (MacOS) is `System Preferences>Dock>Prefer  tabs when opening documents>Always`.

**Comment by adriweb** [comment]
Yeah, looks like GTK is all confused about sizes when the OS makes windows be tabs.
Not sure this is really fixable on our side?

**Comment by debrouxl** [comment]
That sure looks ugly, but I don't know either whether TILP can do anything about it.
It could make sense to test the GTK+3 build ?

**Comment by adriweb** [comment]
Was able to repro on my master build.
Tried it with GTK+3, it forces a new window despite the system tab setting set to Always, so the problem can't even happen. Oh well :)

**Comment by debrouxl** [comment]
Thanks for testing :)
Well, then we can't do much about it with GTK+2.

**Comment by adriweb** [comment]
Well there's probably a way, like calling the resize handler manually or some silly trick like that.
